test/acceptance/coffee/DrainManagerTests.coffee

```diff
@@ -0,0 +1,72 @@
+RealTimeClient = require "./helpers/RealTimeClient"
+FixturesManager = require "./helpers/FixturesManager"
+
+expect = require("chai").expect
+
+async = require "async"
+request = require "request"
+
+Settings = require "settings-sharelatex"
+
+drain = (rate, callback) ->
+	request.post {
+		url: "http://localhost:3026/drain?rate=#{rate}"
+		auth: {
+			user: Settings.internal.realTime.user,
+			pass: Settings.internal.realTime.pass
+		}
+	}, (error, response, data) ->
+		callback error, data
+	return null
+
+describe "DrainManagerTests", ->
+	before (done) ->
+		FixturesManager.setUpProject {
+			privilegeLevel: "owner"
+			project: {
+				name: "Test Project"
+			}
+		}, (e, {@project_id, @user_id}) => done()
```

Comment:
I hate this coffeescript syntax for dereferencing things, but it's where we are. ;-)